### ISSUE TITLE
NoSuchMethodException on Spigot server start-up

### ISSUE THREAD

**Issue by SlimeDog** [lucko/spark#117]
Spigot 1.16.5-3087
spark 1.6.0-b208

On server start-up
```
[14:52:49] [Server thread/INFO]: [spark] Enabling spark v1.6.0
[14:52:49] [Server thread/INFO]: [spark] Using Bukkit scheduler for tick monitoring
[14:52:49] [Server thread/ERROR]: Error occurred while enabling spark v1.6.0 (Is it up to date?)
java.lang.ExceptionInInitializerError: null
        at me.lucko.spark.bukkit.BukkitClassSourceLookup.<clinit>(BukkitClassSourceLookup.java:39) ~[?:?]
        at me.lucko.spark.bukkit.BukkitSparkPlugin.createClassSourceLookup(BukkitSparkPlugin.java:165) ~[?:?]
        at me.lucko.spark.common.SparkPlatform.<init>(SparkPlatform.java:120) ~[?:?]
        at me.lucko.spark.bukkit.BukkitSparkPlugin.onEnable(BukkitSparkPlugin.java:58) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[spigot-1.16.5-3087.jar:3087-Spigot-9fb885e-a0149bf]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[spigot-1.16.5-3087.jar:3087-Spigot-9fb885e-a0149bf]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-1.16.5-3087.jar:3087-Spigot-9fb885e-a0149bf]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:492) ~[spigot-1.16.5-3087.jar:3087-Spigot-9fb885e-a0149bf]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:406) ~[spigot-1.16.5-3087.jar:3087-Spigot-9fb885e-a0149bf]
        at net.minecraft.server.v1_16_R3.MinecraftServer.loadWorld(MinecraftServer.java:554) ~[spigot-1.16.5-3087.jar:3087-Spigot-9fb885e-a0149bf]
        at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:257) ~[spigot-1.16.5-3087.jar:3087-Spigot-9fb885e-a0149bf]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:928) ~[spigot-1.16.5-3087.jar:3087-Spigot-9fb885e-a0149bf]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$0(MinecraftServer.java:273) ~[spigot-1.16.5-3087.jar:3087-Spigot-9fb885e-a0149bf]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NoSuchMethodException: org.bukkit.plugin.java.PluginClassLoader.getPlugin()
        at java.lang.Class.getDeclaredMethod(Class.java:2475) ~[?:?]
        at me.lucko.spark.bukkit.BukkitClassSourceLookup.<clinit>(BukkitClassSourceLookup.java:36) ~[?:?]
        ... 13 more
```
No issue on Paper 1.16.5-750 (equivalent to Spigot release above).

**Comment by Titaniumtown** [comment]
Why are you using spigot?

**Comment by SlimeDog** [comment]
Confirmed fixed in Spark 1.6.0-b210. Thanks.
